workshops/management/commands/report_all_instructor_activity.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import yaml
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+from workshops.models import Award, Badge, Person, Qualification, Role, Task
+
+class Command(BaseCommand):
+    args = ''
+    help = 'Report all instructor activity.'
+
+    instructor_role = Role.objects.get(name='instructor')
+    helper_role = Role.objects.get(name='helper')
+
+    def handle(self, *args, **options):
+        if len(args) != 0:
+            raise CommandError('Usage: report_all_instructor_activity')
+
+        instructor_badge = Badge.objects.get(name='instructor')
+
+        result = []
+        all_awards = Award.objects.filter(badge=instructor_badge)
+        for award in all_awards:
+            person = award.person
+            if not person.email:
```

Comment:
I think we can send this periodic email and add "Do you still don't want to hear from us?", but that's somewhat contrary to what `can_contact` says.